arrows/qt/widgets/LoggerView.h

```diff
@@ -24,8 +25,7 @@ class KWIVER_ALGO_QT_WIDGETS_EXPORT LoggerView : public QWidget
 
 public:
   explicit
-  LoggerView(
-    QWidget* parent = nullptr, Qt::WindowFlags flags = {} );
+  LoggerView( QWidget* parent = nullptr, Qt::WindowFlags flags = {} );
```

Comment:
Not precisely a bug; uncrustify is not configured to remove acceptable but unnecessary line breaks. However, I'm less sure about the added break after `explicit`; is that prescribed/desired? *That* addition might be a bug / missing feature.

Reply:
BTW, newer uncrustify seems to think there does *not* need to be a break after `explicit`. I'm inclined to agree, but what's your take on the intended code style?

Reply:
I think it's fine either way.  I would generally keep in one line unless that line is too long, then breaking after the `explicit` seems like a good way to shorten the line by a few characters.